src/content/docs/journey-performance/slo-tutorial.mdx

```diff
@@ -0,0 +1,296 @@
+        WHERE: entityGuid = '{entityGuid}' AND largestContentfulPaint IS NOT NULL
+        ```
+
+        Where `{entityGuid}` is the service's GUID.
+
+        **Good events fields**
+
+        ```sql
+        FROM: PageViewTiming
+        WHERE: entityGuid = '{entityGuid}' AND largestContentfulPaint < '{largestContentfulPaint}'
+        ```
+
+        * Where `{entityGuid}` is the browser app GUID.
+        * Where `{largestContentfulPaint}` is the amount of time (in milliseconds) to render the largest content element visible in the viewport that you consider provides a good experience for your end user. A frequent standard is 4000 ms.
+
+        To determine a realistic number to use for `{largestContentfulPaint}` in your environment, one typical practice is to select the 95 percentile duration of the responses for the last 7 or 15 days. Find it by using the query builder:
+
+        ```sql
+        SELECT percentile(largestContentfulPaint, 95) FROM PageViewTiming WHERE entityGuid = '{entityGuid}' SINCE 7 days ago LIMIT MAX
+        ```
+    </Collapser>
+
+    <Collapser
+        className="freq-link"
+        id="browser-first-delay"
+        title="Browser app first input delay"
+    >
+        It’s the proportion of page views where the time between a user's first interacion with the page and the time when the browser responds to that interaction is less than a certain threshold.
+
+        **Valid events fields**
+
+        ```sql
+        FROM: PageViewTiming
+        WHERE: entityGuid = '{entityGuid}' AND firstInputDelay IS NOT NULL
+        ```
+
+        Where `{entityGuid}` is the service's GUID.
+
+        **Good events fields**
+
+        ```sql
+        FROM: PageViewTiming
+        WHERE: entityGuid = '{entityGuid}' AND firstInputDelay < {firstInputDelay}
+        ```
+
+        * Where `{entityGuid}` is the browser app GUID.
+        * Where `{firstInputDelay}` is the amount of time (in milliseconds) the browser should respond in to provide a good experience for your end user. A frequent standard is 300 ms.
+
+        To determine a realistic number to use for `{firstInputDelay}` in your environment, one typical practice is to select the 95 percentile duration of the responses for the last 7 or 15 days. Find it by using the query builder:
+
+        ```sql
+        SELECT percentile(firstInputDelay, 95) FROM PageViewTiming WHERE entityGuid = '{entityGuid}' SINCE 7 days ago LIMIT MAX FACET deviceType
+        ```
+    </Collapser>
+
+    <Collapser
+        className="freq-link"
+        id="browser-cumulative-shift"
+        title="Browser app cumulative layout shift"
+    >
+        It’s the proportion of page views with a good cumulative layout shift (CLS). CLS is described as the total sum of all individual layout shift scores for every unexpected layout shift that occurs during the entire lifespan of the page. A layout shift occurs any time a visible element changes its position from one rendered frame to the next.
```

Comment:
So we do have docs for CLS and other core web vitals. I half want to suggest adding in links to the docs (I think they're in browser), but the other half wants to say 0 or only a couple of links to prevent page chains. Just bringing it up to think about.